nix/sources.json

```diff
@@ -24,17 +24,17 @@
         "url_template": "https://github.com/<owner>/<repo>/archive/<rev>.tar.gz"
     },
     "cardano-wallet": {
-        "branch": "refs/tags/v2021-01-28",
+        "branch": "tags/v2020-12-08",
```

Comment:
@rvl why this tag?

Reply:
Oops, that was a messed up rebase from master to develop

Reply:
Fixed